change-beta/@azure-communication-react-ccb6fbb5-9b88-462c-9be8-a99ad73b17b4.json

```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "area": "feature",
+  "workstream": "ImageGallery",
+  "comment": "Add functionality to open image gallery when user taps on an inline image",
```

Comment:
sorry just wanted to double check, we are only showing 1 image per gallery right for this release? Are you going to support multiple images in the gallery later on (similar to teams behaviour)?

Reply:
> sorry just wanted to double check, we are only showing 1 image per gallery right for this release? Are we going to support multiple images in the gallery later on (similar to teams behaviour)?

That's correct. We're supporting one image per gallery for Public Preview. We'll have multiple images support if there's customer asks.